### ISSUE TITLE
Can't load any backgrounds

### ISSUE THREAD

**Issue by DADA30000** [Almamu/linux-wallpaperengine#225]
I'm using NixOS and modified package to use latest commit, but it still doesn't work
It can't display any wallpaper preview except if they're videos.
And it can't attach itself to screen background on Hyprland, even if the wallpaper is a video
[video-wallpaper-screen-root.log](https://github.com/user-attachments/files/15680338/video-wallpaper-screen-root.log)
[example-wallpaper.log](https://github.com/user-attachments/files/15680343/example-wallpaper.log)
[video-wallpaper.log](https://github.com/user-attachments/files/15680345/video-wallpaper.log)

**Comment by Almamu** [comment]
Your assets dir is most likely wrong. Should be ```/home/l0lk3k/.steam/root/steamapps/common/wallpaper_engine/assets```, either that or not specifying an assets folder should work, as the software tries to automatically locate the file

**Comment by DADA30000** [comment]
> Your assets dir is most likely wrong. Should be `/home/l0lk3k/.steam/root/steamapps/common/wallpaper_engine/assets`, either that or not specifying an assets folder should work, as the software tries to automatically locate the file

Yeah, I was testing if I need to set without `assets` at the end, but it doesn't work either way, preview just shows black or gray window and plays sound of a wallpaper for a 1/10 of a second, if I try to preview anything other than a video wallpaper

**Comment by DADA30000** [comment]
And it doesn't work if I don't specify it

**Comment by DADA30000** [comment]
Oh, now it works if I don't specify, and it plays audio, but still grey window

**Comment by DADA30000** [comment]
And I still can't set even a video as background in Hyprland

**Comment by DADA30000** [comment]
Issue is still there

**Comment by DADA30000** [comment]
Ok, I've got it to work and preview scene wallpapers, but I still can't use it as my wallpaper, screen-root option doesn't work, and preview window doesn't have class so I can't use hyprwinwrap
I'm using this rev: e28780562bdf8bcb2867cca7f79b2ed398130eb9
@Almamu

**Comment by DADA30000** [comment]
> [@DADA30000](https://github.com/DADA30000) I'm facing the same issue on hyprland.
> 
> Moreover, I can't even open the preview window.
> 
> I have installed wllpaper engine on steam and I'm sure I have assests in .steam,
> 
> I try to run the command like this
> 
> linux-wallpaperengine 3365981821
> 
> How did you open the preivew window?

this was like half a year ago, I don't remember, and even then, this program is not even available on nix repos now as one of it's dependencies is not maintained and has a lot of CVEs

**Comment by ghost** [comment]
This is an outdated question and can be closed.

**Comment by DADA30000** [comment]
> This is an outdated question and can be closed.

Yep
